Wrappers/Python/test/test_DataContainer.py

```diff
@@ -931,6 +931,170 @@ def test_axpby7(self):
         res = numpy.zeros_like(d1.as_array())
         numpy.testing.assert_array_equal(res, out.as_array())
 
+
+    def test_sapyb_datacontainer_f(self):
```

Comment:
Could you rename the methods:

a) all_DataContainer_floats
b) d1_d2_DataContainer_a_b_float
c) d1_d2_DataContainer_complex_a_b_complex

Reply:
I don't think the proposed naming is clearer. Here there are 3 methods for floats names already:

1. `test_sapyb_scalar_f` for `a` and `b` scalar
2. `test_sapyb_datacontainer_f`  for `a` and `b` `DataContainer`s
3. `test_sapyb_datacontainer_scalar_f` for `a` and `b` mixed

Plus the respective for complex `_c`.